xml_handlers/testing_handler.php

```diff
@@ -175,6 +179,8 @@ public function endElement($parser, $name)
             }
 
             $this->Test->Insert();
+            $build->AddTest($this->Test);
+            $this->Test->SetBuildTest($this->BuildTest);
```

Comment:
Have the BuildTest contain the Test, rather than the Test containing the BuildTest

Reply:
Nevermind, we can reconsider this if/when we clean up the test/buildtest schema